fastplotlib/graphics/image.py

```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
 from typing import Tuple, Any, List
+=======
+from typing import *
```

Comment:
you have a merge conflict that slipped through somehow, just keep this line and delete the ones around it